ci_framework/roles/operator_deploy/README.md

```diff
@@ -5,7 +5,7 @@ Deploy only selected operator(s) on OpenShift
 None
 
 ## Parameters
-* `cifmw_operator_deploy_basedir`: (String) Directory where we will have the RHOL/CRC binary and the configuration (e.g. `artifacts/.rhol_crc_pull_secret.txt`). Default to `cifmw_basedir` which defaults to `~/ci-framework`.
+* `cifmw_operator_deploy_basedir`: (String) Directory where we will have the RHOL/CRC binary and the configuration (e.g. `artifacts/.rhol_crc_pull_secret.txt`). Default to `cifmw_basedir` which defaults to `~/ci-framework-data`.
 * `cifmw_operator_deploy_installyamls`: (String) `install_yamls` root location. Defaults to `cifmw_installyamls_repos` which defaults to `../..`.
```

Comment:
Be aware that the PR that changes this path is still not merged.https://github.com/openstack-k8s-operators/ci-framework/pull/202

Reply:
removed

Reply:
The example config has a cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data" .

Reply:
install_yamls/devsetup/ci-framework/ansible.cfg still pointed to the ~/ci-framework .
After more cleanups the seams to be the right role location in the 

roles_path = ~/ci-framework-data/artifacts/roles:./ci_framework/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles